src/database/migrations/20220601181025-create-user.js

```diff
@@ -30,8 +30,8 @@ module.exports = {
         allowNull: false,
       },
       user_role: {
-        type: Sequelize.ENUM("admin", "requester", "manager", "traveler"),
-        defaultValue: "requester",
+        type: Sequelize.UUID,
+        defaultValue: "7adae2f1-4d35-470d-8512-1b9634330a9e",
```

Comment:
use something like this
## default: Sequelize.fn('uuid_generate_v4'),

Reply:
This is a defaut value of user which is requester role id @habaumugisha1